tests/admin_custom_site/tests.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib import admin
+from django.contrib.admin import sites
+from django.test import SimpleTestCase, override_settings
+
+INSTALLED_APPS = [
+    'admin_custom_site.apps.MyCustomAdminConfig',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+]
+
+
+@override_settings(INSTALLED_APPS=INSTALLED_APPS)
```

Comment:
I think we can omit a temporary variable:
```python
 @override_settings(INSTALLED_APPS=[
    'admin_custom_site.apps.MyCustomAdminConfig',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
])
```

Reply:
Ok